test/src/main/java/org/apache/accumulo/test/MetaGetsReadersIT.java

```diff
@@ -112,6 +113,7 @@ public void test() throws Exception {
     t2.interrupt();
     t1.join();
     t2.join();
+    m.close();
```

Comment:
I didn't realize Mutation was even Closeable :smile:

Reply:
Actually the scanner is called m here let me change that ha.

Reply:
Oh, interesting. I must have misread the code.